### PR TITLE
🚀 Webapp QA deployment using composite actions

### PR DESCRIPTION
**Pull request by rahulpatidar0191**
This action deploy the webapp to QA, it runs only on main branch 

Closes: https://github.com/SatelCreative/DevOps-IT/issues/205

**Tested on** 
1. [ST-PIM](https://github.com/SatelCreative/st-pim/blob/38c2aea0e6bb631797728431059f585a87ccb136/.github/workflows/test-and-deploy-webapp.yml#L100)

- The [action.yml](https://github.com/SatelCreative/satel-qa-deployment/blob/feature/webapp-deployment-shell/action.yml) creates an action
- The [shell script](https://github.com/SatelCreative/satel-qa-vm-deployment/blob/feature/webapp-deployment-shell/DeployToQA.sh) contains the logic
- Readme shows the usage of the actions

### CODE REVIEW

File: DeployToQA.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+APP_NAME=$1
+DOCKER_USER=$2
+DOCKER_PASS=$3
+REGISTRY=$4
+CLEAN_BRANCH_NAME=$5
+
+export CLEAN_BRANCH_NAME=$CLEAN_BRANCH_NAME # set this env variable for docker compose
+
+echo "Deploy to ${APP_NAME}-qa.satelapps.com"
```
Comment (hillairet) [inline DeployToQA.sh]: Should be changed

**Review by hillairet — CHANGES_REQUESTED** [review]

File: DeployToQA.sh (hunk 1 above)
Comment (rahulpatidar0191) [reply DeployToQA.sh]: changed to?
Do you mean to [issue ](https://github.com/SatelCreative/DevOps-IT/issues/240#event-9070909096)you just created?

Comment (hillairet) [reply DeployToQA.sh]: The QA domain is `satelapps.com`???

Comment (rahulpatidar0191) [reply DeployToQA.sh]: Fixed, just copied over from old stuff

**Review by hillairet — APPROVED** [review]

**Pull request merged**